build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -70,9 +70,6 @@
         <BinObj Include="$(SrcDirectory)/**/bin/**/*" />
         <BinObj Include="$(SrcDirectory)/**/obj/**/*" />
 
-        <FilesToRemoveFromStageDirectory Remove="*" />
-        <FilesToRemoveFromStageDirectory Include="$(StageDirectory)/**/*" />
```

Comment:
But isn't it necessary to ensure the stage is clean before we start putting stuff in it?

Maybe calling "RemoveDir" instead will cut down on the spew.